docs/reference_dataset/template_repository/hello-world-root-depls/cf-apps-deployments/generic-app/template/generic-app_manifest-tpl.yml

```diff
@@ -2,3 +2,19 @@ name: generic-app
 instances: (( grab secrets.generic-app.instances ))
 path: .
 
+memory: (( grab secrets.undefined_value || undefined.fallback.value || "64M" )) # as no value are defined, spruce will use the 64M default value
+
+# By default, spruce home dir is deployment directory (ie: credentials-resource/hello-world-root-depls/cf-apps-deployments/generic-app/)
```

Comment:
please consider:
`By default, spruce home dir is in the configuration repo, in the deployment directory (ie: credentials-resource/hello-world-root-depls/cf-apps-deployments/generic-app/)`

Reply:
ok